packages/backend-core/src/configs/configs.ts

```diff
@@ -179,7 +179,7 @@ export async function getGoogleDatasourceConfig(): Promise<
 }
 
 export function getDefaultGoogleConfig(): GoogleInnerConfig | undefined {
-  if (environment.GOOGLE_CLIENT_ID && environment.GOOGLE_CLIENT_SECRET) {
+  if (env.isTest() || (environment.GOOGLE_CLIENT_ID && environment.GOOGLE_CLIENT_SECRET)) {
```

Comment:
Should we not add the values `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` in the test config instead? If it's not there, we will be returning `undefined!`

Reply:
For the gsheets tests I think it's ok given that they are just mocked anyway - but it would be cleaner for sure to do that and not need to add conditional logic to the code here. Will update